reclass/datatypes/parameters.py

```diff
@@ -115,7 +116,7 @@ def _extend_list(self, cur, new, path):
             ret = [cur]
             offset = 1
 
-        for i in xrange(len(new)):
+        for i in range(len(new)):
```

Comment:
`past` isn't an existing dependency so pulling that in might not be ideal, but what do you mean by doing this through builtins?

Personally I think that while xrange does perform, `len(new)` should rarely be so large that it matters.